package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "chess-image-generator",
-  "version": "1.0.9",
+  "version": "1.1.0",
```

Comment:
No, I'm not making it major because of `9 + 1 = 10`, this is a major change since it adds new files.